### DISCUSSION TITLE
Monitor-all to not start automatically?

### DISCUSSION THREAD

**Discussion by dmshimself** [tteck/Proxmox#1495] [General]
I installed monitor-all and as it starts automatically I had several lxc/vm start that had been set to not autostart as options in proxmox, start up as they were indeed not running.  How about not starting the service automatically to allow for excludes to be configured?

**Comment by tteck** [comment]
I think a better option is to automatically skip CT/VMs that have that option set, I'll look at that...

**Comment by dmshimself** [comment]
Or perhaps automatically add excludes for those which are not set to start automatically?  Just a thought

**Reply by tteck** [reply]
Skip instances based on onboot and templates. [8c2a3cc](https://github.com/tteck/Proxmox/commit/8c2a3cc4d774fa13d17f695d6bdf9a4deedb1372)
